.travis.yml

```diff
@@ -1,11 +1,16 @@
+vm:
+  size: large
```

Comment:
This solves our issue for now, but it is costing us in the form of Travis CI credits by using a machine with more memory. We will need to fast-follow to find a more efficient process for building the pages.

Reply:
More info from Travis docs: https://docs.travis-ci.com/user/reference/overview#vm-instance-size